backend/decky_loader/plugin/sandboxed_plugin.py

```diff
@@ -186,6 +186,7 @@ async def on_new_message(self, message : str) -> str|None:
 
         if "uninstall" in data:
             self.uninstalling = data.get("uninstall")
+            return None
```

Comment:
too bad im changing it anyway
```suggestion
            return
```